README.md

```diff
@@ -40,27 +40,32 @@ So that I can see what people are doing
 I want to see all the messages (peeps)
 in a browser
 ```
+![Screenshot](DM_UserStory1.png)
 
 ```
 As a Maker
 So that I can let people know what I am doing  
 I want to post a message (peep) to chitter
 ```
+![Screenshot](DM_UserStory2.png)
 
 ```
 As a Maker
 So that I can see when people are doing things
 I want to see the date the message was posted
 ```
 (Hint the database table will need to change to store the date too)
+![Screenshot](DM_UserStory3.png)
 
 ```
 As a Maker
 So that I can easily see the latest peeps
 I want to see a list of peeps in reverse chronological order
 ```
+![Screenshot](DM_UserStory4.png)
 ```
 As a Maker
 So that I can find relevant peeps
 I want to filter on a specific keyword
 ```
```

Comment:
Your use of diagrams to illustrate the users stories is great! It would also have been nice to see some screenshots of the program in action!

Reply:
I agree, those are some great diagrams! They illustrate your understanding of MVC as well as your thought process very well.